public/apple-app-site-association

```diff
@@ -22,6 +22,22 @@
           {
             "#": "/address/*",
             "comment": "Wallet address"
+          },
+          {
+            "/": "/nfts/asset/*",
+            "comment": "NFT Item"
```

Comment:
should we worry about the overlap with the existing component? Do we need to delete the old?
```
{
   "#": "/nfts/asset/*",
   "comment": "NFT Item"
},

Reply:
we want the overlap, as it will facilitate updating the mobile app for now.